build/ical.js

```diff
@@ -886,7 +894,9 @@ ICAL.design = (function() {
 
   // When adding a value here, be sure to add it to the parameter types!
   var vcardValues = ICAL.helpers.extend(commonValues, {
-
+    component: textType(FROM_ICAL_NEWLINE, TO_ICAL_NEWLINE, "text"),
```

Comment:
In vCard 4.0 semicolon is not escaped in `text` values allowing lists which split on comma. However when there is a need for `lists` of `lists of text` semicolon is used as separator requiring it to be escaped, this value is called `component` in the rfc however the type value of it is still `text`. The name is a little bit misleading because it is a difference in escaping values than specific for newlines the variable can be renamed to something with ESCAPED in it.